custom_components/recycle_app/config_flow.py

```diff
@@ -309,9 +308,9 @@ async def async_step_init(
                     ): str,
                     vol.Optional(
                         "recyclingParkZipCode",
-                        default=self.config_entry.options.get(
-                            "recyclingParkZipCode", ""
-                        ).split("-")[0],
+                        default=str(
+                            next(iter(self.config_entry.options.get("recyclingParkZipCode", [])), "")
+                        ).partition("-")[0],
```

Comment:
`recyclingParkZipCode` option was changed from str to tupple[str, str] by error in v2.7.0 and should be restored to str.

```suggestion
                        default=self.config_entry.options.get(
                            "recyclingParkZipCode", ""
                        ).split("-")[0],
```

We need to fix the root cause as well:
https://github.com/olibos/HomeAssistant-RecycleApp/blob/ac197df62c001d43719c6b05094e87db7981663f/custom_components/recycle_app/config_flow.py#L133-L138

and 
https://github.com/olibos/HomeAssistant-RecycleApp/blob/ac197df62c001d43719c6b05094e87db7981663f/custom_components/recycle_app/config_flow.py#L263-L268
should be changed to:
```python
                if recycling_park_zip_code:
                    zip_code_id, _ = (
                        await self.hass.async_add_executor_job(
                            api.get_zip_code, recycling_park_zip_code, language
                        )
                    )[0]
```
this way we get only the zip code id from the tupple and store it in the option as before v2.7.0